node/signer.ts

```diff
@@ -0,0 +1,36 @@
+import * as crypto from 'crypto';
+import * as elliptic from 'elliptic';
+import {KEYUTIL, b64utohex} from 'jsrsasign';
+
+class EllipticSigner {
```

Comment:
This class will be created in the Node ClientService and be used when requests need signatures.